jane/doc/extensions/modes/reference.md

```diff
@@ -0,0 +1,12 @@
+# Lazy
+`lazy e` contains a thunk that evaluates `e`, as well as a mutable cell to store the
+result of `e`. Upon construction, `lazy e` follows the mode of `e` and the comonadic mode
```

Comment:
I'm not sure what "follows" means here. Do you mean that `lazy e`'s monadic modes are that of `e` and comonadic ones are those of the thunk? But what modes does the thunk have? I think that, in order to be useful, this documentation needs an example -- I don't understand it as written.